src/styles.css

```diff
@@ -10,6 +10,16 @@ main {
   justify-content: space-between;
 }
 
+p {
```

Comment:
We will have to add some class names to other elements that have this p tag

Reply:
Totally agree! Thanks for making a Trello ticket for this!